TestRedditSharp/Program.cs

```diff
@@ -36,7 +36,9 @@ static void Main(string[] args)
                     try
                     {
                         Console.WriteLine("Logging in...");
+#pragma warning disable
```

Comment:
That was because obsolete notifications are annoying output, i'll change it to be that specific warning right now